website/docs/downloads/windows.md

```diff
@@ -25,6 +25,21 @@ the changes and improvements introduced by this release.
     If Windows 8.x or Windows 10 prevents you from running DOSBox Staging via
     Microsoft Defender SmartScreen, see the [section below](#microsoft-defender-smartscreen).
 
+For 32-bit Windows 7 and newer, and for 64-bit Windows 7 hosts use the respective MSVC builds:
```

Comment:
Please don't use the word "host". That makes sense for VMs to remove ambiguity, but it's not needed here and only muddies the waters (e.g. "I'm running a Linux guest in this Linux VirtualBox host").